base/applications/cmdutils/reg/lang/bg-BG.rc

```diff
@@ -2,42 +2,180 @@ LANGUAGE LANG_BULGARIAN, SUBLANG_DEFAULT
 
 STRINGTABLE
 {
-    STRING_USAGE, "Usage:\n  REG [operation] [parameters]\n\nSupported operations:\n  ADD | DELETE | IMPORT | EXPORT | QUERY | SAVE | RESTORE\n\nFor help on a specific operation, type:\n  REG [operation] /?\n\n"
-    STRING_ADD_USAGE, "REG ADD ключ_име [/v стойност_име | /ve] [/t вид] [/s разделител] [/d данни] [/f]\n"
-    STRING_DELETE_USAGE, "REG DELETE ключ_име [/v стойност_име | /ve | /va] [/f]\n"
-    STRING_QUERY_USAGE, "REG QUERY ключ_име [/v стойност_име | /ve] [/s]\n"
+    STRING_USAGE, "Usage:\n\
+  REG [operation] [parameters]\n\n\
+Supported operations:\n\
+  ADD | COPY | DELETE | EXPORT | IMPORT | QUERY\n\n\
+For help on a specific operation, type:\n\
+  REG [operation] /?\n\n"
+
+    STRING_ADD_USAGE, "REG ADD <key> [/v value_name | /ve] [/t type] [/s separator] [/d data] [/f]\n\n\
+  Adds a key to the registry or adds a new value to a given registry key.\n\n\
```

Comment:
Is this expected that the beginning of these help lines don't start right at the left margin?

Reply:
It nearly the same as the wine code. Only missing "\", it leads to compile errors. I tried to modify only the minimum, but it can be changed, if wanted/needed. I will double check how these help lines are display in console.

Reply:
- Console output and it looks good. 
![Screenshot from 2022-03-10 20-12-03](https://user-images.githubusercontent.com/46087964/157739572-e73d78a1-feba-4d69-a25f-15378032062e.png)
- en-US.rc and reg.rc are the reference in reactos and wine. I tried to keep differences at a minimum, because it is a sync.